src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -698,10 +698,13 @@ export default class QuestionnaireForm extends Component {
     var qr = window.LForms.Util.getFormFHIRData('QuestionnaireResponse', 'R4', "#formContainer");
     console.log(qr);
     qr.status = status;
-    qr.author = {
-      reference:
-        this.getPractitioner()
-    };
+    if (this.getPractitioner() !== "Unknown") {
```

Comment:
What do we do if this.getPracticitioner() is "Unknown"?  qr.author.reference will not be set?

Reply:
yes it's not set at all